setup.py

```diff
@@ -13,10 +13,10 @@
 
 
 INSTALL_REQUIRES = [
-    "transformers == 4.35.0",
+    "transformers == 4.36.2",
     "accelerate == 0.23.0",
     "optimum >= 1.14.0",
-    "huggingface_hub >= 0.14.0",
+    "huggingface_hub >= 0.19.4",
```

Comment:
```suggestion
    "huggingface_hub >= 0.20.1",
```
I think it'll be needed for us to use the `huggingface_hub.get_token()` function, which is the recommended way to ge the value of the token from now on.